### PR TITLE
minor glove refactor

### PR DESCRIPTION
**Pull request by TDSSS**
## What Does This PR Do
PR now just contains a slight clean-up of shitty old format.

I also fixed some formatting in the gloves file that I noticed while there, bringing it in line with our code standards.

## Why It's Good For The Game
Just cleans up some confusing formatting, makes searching for those gloves easier

## Changelog
:cl:
tweak: Just slight gloves refactor, shouldn't impact anything in game.
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
I really don't think doing this level of checks is warranted enough just to blacklist rings (and maybe bracers).

Fingerless gloves and latex gloves would absolutely be able to handle nettles.

**Comment by TDSSS** [comment]
But would they be able to handle **death nettles**?

**Comment by variableundefined** [comment]
Not much of a fan of this specific check myself either.

**Comment by TDSSS** [comment]
Kay, I'll remove it and turn this into just a minor glove refactor to make it less shitcode

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -139,8 +139,9 @@
 	icon_state = "rainbow"
 	item_state = "rainbowgloves"
 	item_color = "rainbow"
-	clown
```
Comment (variableundefined) [inline code/modules/clothing/gloves/color.dm]: RELATIVE PATHING WHAT IN TARNATION

File: code/modules/clothing/gloves/color.dm [hunk 2]
```diff
@@ -170,11 +171,11 @@
 	item_state = "graygloves"
 	item_color="grey"
 
-	rd
-		item_color = "director"			//Exists for washing machines. Is not different from gray gloves in any way.
+/obj/item/clothing/gloves/color/grey/rd
+	item_color = "director"			//Exists for washing machines. Is not different from gray gloves in any way.
 
-	hop
```
Comment (variableundefined) [inline code/modules/clothing/gloves/color.dm]: AAAAAAAAA

File: code/modules/clothing/gloves/color.dm [hunk 3]
```diff
@@ -219,8 +220,8 @@
 	item_state = "wgloves"
 	item_color="mime"
 
-	redcoat
```
Comment (variableundefined) [inline code/modules/clothing/gloves/color.dm]: FRIC

**Pull request merged**